docs/UserGuide.md

```diff
@@ -146,25 +146,68 @@ Examples:
 * `cfind John` returns `john` and `Johnathon Doe`
 * `cfind alex david` returns `Alex Yeoh` and `David Li`
 
-### Deleting a person : `delete`
+### Editing an event : `eedit`
 
-Deletes the specified person from the address book.
+Edits an existing event in the event list.
 
-Format: `delete INDEX`
+Format: `eedit INDEX [n/NAME] [at/START_TIME] [end/END_TIME] [d/DESCRIPTION] [a/ADDRESS] [z/ZOOM] [dt/TAG_DELETED]…​ [t/TAG_ADDED]…​`
 
-* Deletes the person at the specified `INDEX`.
-* The index refers to the index number shown in the displayed person list.
-* The index **must be a positive integer** 1, 2, 3, …​
+
+:information_source: **Note:** Start time should be of format “dd-MM-yyyy HH:mm” (date-month-year Hour:minutes in 24 hr format).
```

Comment:
```suggestion
<div markdown="span" class="alert alert-info">
:information_source: **Note:** Start time should be of format “dd-MM-yyyy HH:mm” (date-month-year Hour:minutes in 24 hr format).
</div>
```

Wrap in a div to highlight this note.

Reply:
Need 1 line spacing between the text and div tags, otherwise the div does not show, i have added the spacing before and after the text in line 159.